environments/edusources/acceptance/invoke.yml

```diff
@@ -4,7 +4,7 @@
 env: "acceptance"
 
 django:
-  debug: true
+  debug: false
```

Comment:
en dit?

Reply:
Zal ook af en toe wisselen afhankelijk van wat de vraag is.